x-pack/plugins/enterprise_search/public/applications/workplace_search/assets/connection_illustration.svg

```diff
@@ -0,0 +1,263 @@
+<svg width="398" height="552" viewBox="0 0 398 552" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should minify this image.

Reply:
```
<svg width="398" height="552" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill="#22272D" d="M0 0h398v552H0z"/><mask id="a" mask-type="alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="398" height="552"><path d="M0 4a4 4 0 014-4h394v552H4a4 4 0 01-4-4V4z" fill="#22272D"/></mask><g mask="url(#a)"><path d="M197 471.611l89.25 48.346L197 568.303l-89.25-48.346L197 471.611z" fill="#171A1F"/><path d="M197 471.611l89.25 48.346L197 568.303v-96.692z" fill="#2E353D"/><path d="M245.75 445.904L335 494.25l-48.75 25.846L197 471.75l48.75-25.846zM148.25 445.904L59 494.25l48.75 25.846L197 471.75l-48.75-25.846z" fill="#3D434B"/><g filter="url(#filter0_d)"><path d="M131 409h105c13.807 0 25 11.193 25 25v20H156c-13.807 0-25-11.193-25-25v-20z" fill="#696F76"/></g><g filter="url(#filter1_d)"><path d="M131 462h105c13.807 0 25 11.193 25 25v20H156c-13.807 0-25-11.193-25-25v-20z" fill="#696F76"/></g><g filter="url(#filter2_d)"><path d="M131 515h105c13.807 0 25 11.193 25 25v20H156c-13.807 0-25-11.193-25-25v-20z" fill="#696F76"/></g><g filter="url(#filter3_d)" fill="#fff"><path d="M167.835 428.192h2.505c-.036-2.203-1.847-3.725-4.6-3.725-2.711 0-4.697 1.498-4.685 3.743-.006 1.823 1.28 2.868 3.369 3.369l1.346.338c1.346.326 2.095.712 2.101 1.545-.006.906-.863 1.522-2.192 1.522-1.358 0-2.336-.628-2.42-1.866h-2.53c.067 2.675 1.98 4.057 4.981 4.057 3.018 0 4.793-1.443 4.799-3.707-.006-2.058-1.558-3.151-3.707-3.634l-1.111-.265c-1.074-.248-1.974-.646-1.955-1.534 0-.797.706-1.382 1.986-1.382 1.249 0 2.016.567 2.113 1.539zM176.329 437.181c2.813 0 4.563-1.926 4.563-4.781 0-2.874-1.75-4.793-4.563-4.793-2.814 0-4.564 1.919-4.564 4.793 0 2.855 1.75 4.781 4.564 4.781zm.012-1.992c-1.298 0-1.962-1.189-1.962-2.807 0-1.618.664-2.813 1.962-2.813 1.273 0 1.937 1.195 1.937 2.813s-.664 2.807-1.937 2.807zM188.431 433.052c.006 1.243-.845 1.895-1.769 1.895-.972 0-1.6-.682-1.606-1.774v-5.446h-2.571v5.904c.006 2.168 1.273 3.49 3.139 3.49 1.394 0 2.397-.719 2.813-1.805h.097V437h2.469v-9.273h-2.572v5.325zM192.98 437h2.572v-5.246c0-1.141.833-1.926 1.968-1.926.356 0 .845.061 1.086.139v-2.282a3.876 3.876 0 00-.809-.091c-1.038 0-1.889.604-2.227 1.751h-.097v-1.618h-2.493V437zM203.614 437.181c2.469 0 4.021-1.449 4.141-3.58h-2.427c-.15.99-.802 1.546-1.684 1.546-1.201 0-1.98-1.008-1.98-2.783 0-1.751.785-2.753 1.98-2.753.942 0 1.546.622 1.684 1.545h2.427c-.108-2.143-1.732-3.549-4.153-3.549-2.813 0-4.552 1.949-4.552 4.793 0 2.819 1.708 4.781 4.564 4.781zM213.548 437.181c2.294 0 3.839-1.117 4.201-2.837l-2.378-.157c-.26.706-.924 1.074-1.781 1.074-1.286 0-2.101-.851-2.101-2.233v-.006h6.315v-.707c0-3.151-1.908-4.708-4.359-4.708-2.729 0-4.497 1.937-4.497 4.799 0 2.94 1.744 4.775 4.6 4.775zm-2.059-5.753c.054-1.057.857-1.902 1.998-1.902 1.117 0 1.89.797 1.896 1.902h-3.894zM225.436 437l.918-2.819h4.461l.917 2.819h2.802l-4.263-12.364h-3.368L222.635 437h2.801zm1.582-4.86l1.521-4.678h.096l1.522 4.678h-3.139zM168.573 481.192h2.506c-.037-2.203-1.848-3.725-4.6-3.725-2.711 0-4.697 1.498-4.685 3.743-.006 1.823 1.28 2.868 3.368 3.369l1.347.338c1.346.326 2.095.712 2.101 1.545-.006.906-.864 1.522-2.192 1.522-1.358 0-2.336-.628-2.421-1.866h-2.529c.066 2.675 1.98 4.057 4.98 4.057 3.019 0 4.794-1.443 4.8-3.707-.006-2.058-1.558-3.151-3.707-3.634l-1.111-.265c-1.074-.248-1.974-.646-1.956-1.534 0-.797.707-1.382 1.986-1.382 1.25 0 2.017.567 2.113 1.539zM177.067 490.181c2.814 0 4.564-1.926 4.564-4.781 0-2.874-1.75-4.793-4.564-4.793-2.813 0-4.564 1.919-4.564 4.793 0 2.855 1.751 4.781 4.564 4.781zm.012-1.992c-1.298 0-1.962-1.189-1.962-2.807 0-1.618.664-2.813 1.962-2.813 1.274 0 1.938 1.195 1.938 2.813s-.664 2.807-1.938 2.807zM189.17 486.052c.006 1.243-.845 1.895-1.769 1.895-.972 0-1.6-.682-1.606-1.774v-5.446h-2.572v5.904c.007 2.168 1.274 3.49 3.14 3.49 1.394 0 2.396-.719 2.813-1.805h.096V490h2.47v-9.273h-2.572v5.325zM193.719 490h2.571v-5.246c0-1.141.833-1.926 1.968-1.926.357 0 .846.061 1.087.139v-2.282a3.876 3.876 0 00-.809-.091c-1.038 0-1.889.604-2.227 1.751h-.097v-1.618h-2.493V490zM204.353 490.181c2.469 0 4.02-1.449 4.141-3.58h-2.427c-.151.99-.803 1.546-1.684 1.546-1.201 0-1.98-1.008-1.98-2.783 0-1.751.785-2.753 1.98-2.753.942 0 1.545.622 1.684 1.545h2.427c-.109-2.143-1.733-3.549-4.153-3.549-2.814 0-4.552 1.949-4.552 4.793 0 2.819 1.708 4.781 4.564 4.781zM214.286 490.181c2.295 0 3.84-1.117 4.202-2.837l-2.378-.157c-.26.706-.924 1.074-1.781 1.074-1.286 0-2.101-.851-2.101-2.233v-.006h6.315v-.707c0-3.151-1.908-4.708-4.359-4.708-2.729 0-4.498 1.937-4.498 4.799 0 2.94 1.745 4.775 4.6 4.775zm-2.058-5.753c.054-1.057.857-1.902 1.998-1.902 1.117 0 1.89.797 1.896 1.902h-3.894zM224.044 490h5.282c2.777 0 4.256-1.449 4.256-3.417 0-1.829-1.31-2.934-2.717-3v-.121c1.286-.29 2.234-1.201 2.234-2.656 0-1.854-1.376-3.17-4.105-3.17h-4.95V490zm2.614-2.137v-3.284h2.191c1.256 0 2.034.724 2.034 1.745 0 .929-.633 1.539-2.094 1.539h-2.131zm0-5.053v-3.061h1.986c1.159 0 1.817.598 1.817 1.491 0 .978-.797 1.57-1.866 1.57h-1.937zM167.801 533.192h2.506c-.037-2.203-1.848-3.725-4.6-3.725-2.711 0-4.697 1.498-4.685 3.743-.006 1.823 1.28 2.868 3.369 3.369l1.346.338c1.346.326 2.095.712 2.101 1.545-.006.906-.864 1.522-2.192 1.522-1.358 0-2.336-.628-2.421-1.866h-2.529c.066 2.675 1.98 4.057 4.98 4.057 3.019 0 4.794-1.443 4.8-3.707-.006-2.058-1.558-3.151-3.707-3.634l-1.111-.265c-1.074-.248-1.974-.646-1.956-1.534 0-.797.707-1.382 1.986-1.382 1.25 0 2.017.567 2.113 1.539zM176.295 542.181c2.814 0 4.564-1.926 4.564-4.781 0-2.874-1.75-4.793-4.564-4.793-2.813 0-4.564 1.919-4.564 4.793 0 2.855 1.751 4.781 4.564 4.781zm.012-1.992c-1.298 0-1.962-1.189-1.962-2.807 0-1.618.664-2.813 1.962-2.813 1.274 0 1.938 1.195 1.938 2.813s-.664 2.807-1.938 2.807zM188.398 538.052c.006 1.243-.845 1.895-1.769 1.895-.972 0-1.6-.682-1.606-1.774v-5.446h-2.572v5.904c.007 2.168 1.274 3.49 3.14 3.49 1.394 0 2.396-.719 2.813-1.805h.096V542h2.47v-9.273h-2.572v5.325zM192.947 542h2.571v-5.246c0-1.141.834-1.926 1.968-1.926.357 0 .846.061 1.087.139v-2.282a3.876 3.876 0 00-.809-.091c-1.038 0-1.889.604-2.227 1.751h-.097v-1.618h-2.493V542zM203.581 542.181c2.469 0 4.02-1.449 4.141-3.58h-2.427c-.151.99-.803 1.546-1.684 1.546-1.201 0-1.98-1.008-1.98-2.783 0-1.751.785-2.753 1.98-2.753.942 0 1.545.622 1.684 1.545h2.427c-.109-2.143-1.733-3.549-4.153-3.549-2.814 0-4.552 1.949-4.552 4.793 0 2.819 1.708 4.781 4.564 4.781zM213.514 542.181c2.295 0 3.84-1.117 4.202-2.837l-2.378-.157c-.26.706-.924 1.074-1.781 1.074-1.286 0-2.101-.851-2.101-2.233v-.006h6.315v-.707c0-3.151-1.908-4.708-4.359-4.708-2.729 0-4.498 1.937-4.498 4.799 0 2.94 1.745 4.775 4.6 4.775zm-2.058-5.753c.054-1.057.857-1.902 1.998-1.902 1.117 0 1.89.797 1.896 1.902h-3.894zM234.18 533.965c-.344-2.856-2.505-4.498-5.366-4.498-3.266 0-5.76 2.306-5.76 6.351 0 4.033 2.451 6.351 5.76 6.351 3.169 0 5.083-2.107 5.366-4.383l-2.644-.012c-.247 1.322-1.286 2.083-2.68 2.083-1.878 0-3.151-1.395-3.151-4.039 0-2.572 1.255-4.039 3.169-4.039 1.431 0 2.463.828 2.662 2.186h2.644z"/></g><path d="M197 568.172l89.25 48.345L197 664.863l-89.25-48.346L197 568.172z" fill="#171A1F"/><path d="M197 568.171l89.25 48.346L197 664.863v-96.692z" fill="#22272D"/><path d="M286.25 519.898L197 568.244l48.75 25.846L335 545.744l-48.75-25.846zM107.75 519.898L197 568.244l-48.75 25.846L59 545.744l48.75-25.846z" fill="#3D434B"/><g filter="url(#filter4_d)"><path fill="#DBE4EE" d="M185.037 259.776h26.925v30.056h-26.925z"/></g><g filter="url(#filter5_d)"><path d="M241.13 289.409l-42.409 42.409-42.409-42.409h84.818z" fill="#fff"/></g><path opacity=".14" d="M210.207 320.332L183.039 347.5l-27.169-27.168h54.337z" fill="#fff"/><g filter="url(#filter6_d)"><path d="M233.841 311.276l-15.131 15.13-15.13-15.13h30.261z" fill="#696F76"/></g><path d="M328 75c0 68.483-55.517 124-124 124S80 143.483 80 75 135.517-49 204-49 328 6.517 328 75z" fill="url(#paint0_linear)"/><g filter="url(#filter7_d)"><path d="M186 219c-68.483 0-124-55.517-124-124h124v124z" fill="#00C8AC"/></g><g filter="url(#filter8_d)"><path d="M294 75c0 48.601-39.399 88-88 88V75h88z" fill="#FFB92F"/></g><g filter="url(#filter9_dd)"><path fill="#fff" d="M38 163h134v33H38z"/></g><g filter="url(#filter10_dd)"><path fill="#fff" d="M225 97h134v33H225z"/></g><path d="M293.567 117.636h1.726l-.157-2.876 2.415 1.584.867-1.506-2.586-1.293 2.586-1.292-.867-1.506-2.415 1.584.157-2.876h-1.726l.163 2.876-2.415-1.584-.866 1.506 2.578 1.292-2.578 1.293.866 1.506 2.415-1.584-.163 2.876zM304.926 117.636h1.726l-.156-2.876 2.415 1.584.866-1.506-2.585-1.293 2.585-1.292-.866-1.506-2.415 1.584.156-2.876h-1.726l.164 2.876-2.415-1.584-.867 1.506 2.578 1.292-2.578 1.293.867 1.506 2.415-1.584-.164 2.876zM316.286 117.636h1.725l-.156-2.876 2.415 1.584.866-1.506-2.585-1.293 2.585-1.292-.866-1.506-2.415 1.584.156-2.876h-1.725l.163 2.876-2.415-1.584-.866 1.506 2.578 1.292-2.578 1.293.866 1.506 2.415-1.584-.163 2.876zM327.645 117.636h1.726l-.156-2.876 2.414 1.584.867-1.506-2.585-1.293 2.585-1.292-.867-1.506-2.414 1.584.156-2.876h-1.726l.163 2.876-2.414-1.584-.867 1.506 2.578 1.292-2.578 1.293.867 1.506 2.414-1.584-.163 2.876zM339.004 117.636h1.726l-.156-2.876 2.415 1.584.866-1.506-2.585-1.293 2.585-1.292-.866-1.506-2.415 1.584.156-2.876h-1.726l.164 2.876-2.415-1.584-.867 1.506 2.579 1.292-2.579 1.293.867 1.506 2.415-1.584-.164 2.876z" fill="#FFB92F"/><path d="M242.408 111.746h2.064c-.03-1.814-1.522-3.067-3.789-3.067-2.232 0-3.868 1.233-3.858 3.082-.005 1.502 1.054 2.362 2.774 2.775l1.109.278c1.109.268 1.725.587 1.73 1.273-.005.745-.711 1.252-1.804 1.252-1.119 0-1.924-.517-1.994-1.536h-2.083c.055 2.203 1.631 3.341 4.101 3.341 2.486 0 3.948-1.188 3.953-3.052-.005-1.696-1.283-2.596-3.053-2.993l-.914-.219c-.885-.204-1.626-.532-1.611-1.263 0-.656.581-1.138 1.635-1.138 1.03 0 1.661.467 1.74 1.267zM249.419 119.149c1.889 0 3.162-.92 3.46-2.337l-1.958-.129c-.214.582-.761.885-1.467.885-1.059 0-1.73-.701-1.73-1.839v-.005h5.2v-.582c0-2.595-1.571-3.878-3.589-3.878-2.247 0-3.704 1.596-3.704 3.953 0 2.421 1.437 3.932 3.788 3.932zm-1.695-4.738c.045-.87.706-1.566 1.645-1.566.92 0 1.557.656 1.562 1.566h-3.207zM257.677 119.149c2.033 0 3.311-1.193 3.41-2.948h-1.999c-.124.815-.661 1.273-1.387 1.273-.989 0-1.63-.831-1.63-2.292 0-1.442.646-2.267 1.63-2.267.776 0 1.273.512 1.387 1.272h1.999c-.089-1.764-1.427-2.923-3.42-2.923-2.317 0-3.749 1.606-3.749 3.948 0 2.321 1.407 3.937 3.759 3.937zM262.363 119h2.118v-4.32c0-.94.686-1.586 1.621-1.586.293 0 .696.049.895.114v-1.879a3.168 3.168 0 00-.667-.075c-.855 0-1.556.497-1.834 1.442h-.08v-1.332h-2.053V119zM271.136 119.149c1.889 0 3.162-.92 3.46-2.337l-1.958-.129c-.214.582-.761.885-1.467.885-1.059 0-1.73-.701-1.73-1.839v-.005h5.2v-.582c0-2.595-1.571-3.878-3.589-3.878-2.247 0-3.704 1.596-3.704 3.953 0 2.421 1.437 3.932 3.788 3.932zm-1.695-4.738c.045-.87.706-1.566 1.645-1.566.92 0 1.557.656 1.562 1.566h-3.207zM280.005 111.364h-1.437v-1.83h-2.118v1.83h-1.044v1.591h1.044v3.977c-.01 1.496 1.01 2.237 2.546 2.172a4.113 4.113 0 001.148-.198l-.333-1.576c-.104.019-.328.069-.527.069-.422 0-.716-.159-.716-.746v-3.698h1.437v-1.591zM282.543 113.785a1.21 1.21 0 001.193-1.193 1.208 1.208 0 00-1.193-1.184 1.189 1.189 0 100 2.377zm0 5.344a1.21 1.21 0 001.193-1.193 1.208 1.208 0 00-1.193-1.183 1.188 1.188 0 100 2.376z" fill="#343741"/><path d="M85.057 184h1.057v-3.409h1.909c.076 0 .149 0 .221-.004L90.085 184h1.227l-1.973-3.605c1.113-.379 1.633-1.287 1.633-2.446 0-1.543-.92-2.676-2.966-2.676h-2.95V184zm1.057-4.364v-3.426h1.858c1.414 0 1.96.691 1.96 1.739s-.546 1.687-1.943 1.687h-1.875zM93.233 184h3.153c2.063 0 2.915-1.006 2.915-2.318 0-1.381-.955-2.131-1.756-2.182v-.085c.75-.205 1.415-.699 1.415-1.824 0-1.279-.852-2.318-2.676-2.318h-3.052V184zm1.056-.938v-3.034h2.148c1.142 0 1.858.767 1.858 1.654 0 .767-.528 1.38-1.909 1.38h-2.097zm0-3.954v-2.898h1.995c1.159 0 1.67.614 1.67 1.381 0 .92-.75 1.517-1.704 1.517h-1.96zM104.282 184.119c1.764.022 2.914-1.308 2.914-2.932 0-1.687-1.248-2.897-2.744-2.897-.899 0-1.67.439-2.148 1.159h-.068c.004-2.144.831-3.375 2.182-3.375.92 0 1.474.579 1.67 1.381h1.04c-.213-1.373-1.244-2.319-2.71-2.319-1.965 0-3.171 1.739-3.171 4.841 0 3.226 1.603 4.125 3.035 4.142zm0-.937c-1.096 0-1.897-.933-1.892-1.977.004-1.044.843-1.978 1.926-1.978 1.069 0 1.858.878 1.858 1.96 0 1.108-.823 1.995-1.892 1.995zM112.151 175.136c-1.773-.017-2.915 1.313-2.915 2.932 0 1.688 1.244 2.898 2.744 2.898.903 0 1.671-.443 2.148-1.159h.068c0 2.148-.835 3.375-2.182 3.375-.92 0-1.483-.58-1.67-1.381h-1.04c.204 1.381 1.244 2.318 2.71 2.318 1.96 0 3.171-1.738 3.171-4.841 0-3.221-1.603-4.125-3.034-4.142zm0 .938c1.09 0 1.892.937 1.892 1.977s-.853 1.977-1.927 1.977c-1.073 0-1.858-.869-1.858-1.96 0-1.108.819-1.994 1.893-1.994zM117.514 184h3.153c2.063 0 2.915-1.006 2.915-2.318 0-1.381-.955-2.131-1.756-2.182v-.085c.75-.205 1.415-.699 1.415-1.824 0-1.279-.852-2.318-2.676-2.318h-3.051V184zm1.057-.938v-3.034h2.147c1.142 0 1.858.767 1.858 1.654 0 .767-.528 1.38-1.909 1.38h-2.096zm0-3.954v-2.898h1.994c1.159 0 1.67.614 1.67 1.381 0 .92-.75 1.517-1.704 1.517h-1.96zM128.392 184.119c1.662 0 2.881-1.227 2.881-2.897 0-1.692-1.176-2.932-2.778-2.932-.588 0-1.159.209-1.517.494h-.052l.307-2.574h3.648v-.937h-4.534l-.529 4.295.989.119c.362-.259.98-.447 1.517-.443 1.112.009 1.926.853 1.926 1.995 0 1.12-.784 1.943-1.858 1.943-.895 0-1.606-.575-1.687-1.364h-1.023c.064 1.33 1.21 2.301 2.71 2.301zM133.977 175.273h-1.108l3.205 8.727h1.091l3.204-8.727h-1.108l-2.591 7.346h-.102l-2.591-7.346zM147.784 178h1.091c-.324-1.688-1.738-2.847-3.579-2.847-2.237 0-3.818 1.722-3.818 4.483 0 2.762 1.568 4.483 3.886 4.483 2.079 0 3.562-1.385 3.562-3.545v-.938h-3.323v.938h2.301c-.03 1.568-1.061 2.557-2.54 2.557-1.619 0-2.864-1.228-2.864-3.495 0-2.267 1.245-3.494 2.796-3.494 1.261 0 2.118.712 2.488 1.858z" fill="#00B8AA"/><path d="M63.038 174.818h-2.153V185h2.153v-10.182zM68.338 185c3.102 0 4.982-1.919 4.982-5.101 0-3.172-1.88-5.081-4.952-5.081h-3.64V185h3.61zm-1.457-1.844v-6.493h1.373c1.909 0 2.918.974 2.918 3.236 0 2.272-1.01 3.257-2.923 3.257H66.88zM76.032 178.825a1.21 1.21 0 001.194-1.193 1.208 1.208 0 00-1.194-1.183 1.188 1.188 0 100 2.376zm0 4.992a1.21 1.21 0 001.194-1.193 1.208 1.208 0 00-1.194-1.184 1.188 1.188 0 100 2.377z" fill="#343741"/><g filter="url(#filter11_d)" fill="#fff"><path d="M96.455 118.492c-.364-3.023-2.653-4.762-5.682-4.762-3.459 0-6.098 2.442-6.098 6.725 0 4.269 2.595 6.724 6.098 6.724 3.355 0 5.382-2.231 5.682-4.641l-2.8-.012c-.262 1.399-1.361 2.205-2.838 2.205-1.988 0-3.336-1.477-3.336-4.276 0-2.723 1.33-4.277 3.356-4.277 1.514 0 2.607.876 2.818 2.314h2.8zM102.745 127.192c2.978 0 4.832-2.039 4.832-5.063 0-3.042-1.854-5.075-4.832-5.075-2.979 0-4.833 2.033-4.833 5.075 0 3.024 1.854 5.063 4.833 5.063zm.013-2.11c-1.375 0-2.078-1.259-2.078-2.972s.703-2.979 2.078-2.979c1.348 0 2.051 1.266 2.051 2.979s-.703 2.972-2.051 2.972zM111.991 121.324c.006-1.266.76-2.007 1.86-2.007 1.093 0 1.751.716 1.745 1.917V127h2.723v-6.251c0-2.289-1.342-3.695-3.388-3.695-1.457 0-2.512.716-2.953 1.86h-.115v-1.732h-2.595V127h2.723v-5.676zM125.665 117.182h-1.95v-.659c0-.664.269-1.054 1.068-1.054.326 0 .658.07.875.14l.48-2.045a7.182 7.182 0 00-1.924-.269c-1.847 0-3.222 1.042-3.222 3.177v.71h-1.387v2.045h1.387V127h2.723v-7.773h1.95v-2.045zM127.248 127h2.723v-9.818h-2.723V127zm1.368-11.084c.812 0 1.477-.62 1.477-1.38 0-.755-.665-1.375-1.477-1.375-.805 0-1.47.62-1.47 1.375 0 .76.665 1.38 1.47 1.38zM136.496 130.822c2.806 0 4.8-1.278 4.8-3.726v-9.914h-2.703v1.649h-.103c-.364-.799-1.163-1.777-2.806-1.777-2.154 0-3.976 1.675-3.976 5.018 0 3.266 1.771 4.787 3.983 4.787 1.566 0 2.441-.786 2.799-1.598h.115v1.797c0 1.284-.863 1.808-2.045 1.808-1.202 0-1.809-.524-2.033-1.054l-2.518.339c.326 1.483 1.841 2.671 4.487 2.671zm.058-6.008c-1.336 0-2.065-1.061-2.065-2.755 0-1.668.716-2.838 2.065-2.838 1.323 0 2.064 1.118 2.064 2.838 0 1.732-.754 2.755-2.064 2.755zM149.679 122.82c.007 1.316-.894 2.007-1.872 2.007-1.03 0-1.694-.723-1.701-1.88v-5.765h-2.723v6.251c.007 2.295 1.349 3.695 3.324 3.695 1.477 0 2.538-.761 2.979-1.911h.102V127h2.614v-9.818h-2.723v5.638zM154.501 127h2.723v-5.555c0-1.208.882-2.039 2.083-2.039.378 0 .895.064 1.151.147v-2.416a4.126 4.126 0 00-.857-.096c-1.099 0-2 .639-2.358 1.854h-.103v-1.713h-2.639V127zM165.803 127.192c2.429 0 4.066-1.183 4.449-3.004l-2.518-.167c-.275.748-.978 1.138-1.886 1.138-1.362 0-2.224-.901-2.224-2.365v-.006h6.686v-.748c0-3.337-2.02-4.986-4.615-4.986-2.89 0-4.763 2.052-4.763 5.082 0 3.113 1.848 5.056 4.871 5.056zm-2.179-6.092c.057-1.118.907-2.013 2.115-2.013 1.183 0 2.001.843 2.007 2.013h-4.122zM131.793 139.492c-.364-3.023-2.652-4.762-5.682-4.762-3.458 0-6.098 2.442-6.098 6.725 0 4.269 2.595 6.724 6.098 6.724 3.356 0 5.382-2.231 5.682-4.641l-2.799-.012c-.263 1.399-1.362 2.205-2.839 2.205-1.987 0-3.336-1.477-3.336-4.276 0-2.723 1.329-4.277 3.356-4.277 1.515 0 2.608.876 2.819 2.314h2.799zM136.37 134.909h-2.723V148h2.723v-13.091zM138.471 148h2.723v-9.818h-2.723V148zm1.368-11.084c.812 0 1.477-.62 1.477-1.38 0-.755-.665-1.375-1.477-1.375-.805 0-1.47.62-1.47 1.375 0 .76.665 1.38 1.47 1.38zM147.77 148.192c2.429 0 4.065-1.183 4.449-3.004l-2.519-.167c-.275.748-.978 1.138-1.885 1.138-1.362 0-2.225-.901-2.225-2.365v-.006h6.686v-.748c0-3.337-2.02-4.986-4.615-4.986-2.889 0-4.762 2.052-4.762 5.082 0 3.113 1.847 5.056 4.871 5.056zm-2.18-6.092c.058-1.118.908-2.013 2.116-2.013 1.182 0 2.001.843 2.007 2.013h-4.123zM156.696 142.324c.007-1.266.761-2.007 1.86-2.007 1.093 0 1.752.716 1.745 1.917V148h2.723v-6.251c0-2.289-1.342-3.695-3.387-3.695-1.458 0-2.512.716-2.954 1.86h-.115v-1.732h-2.595V148h2.723v-5.676zM170.313 138.182h-1.848v-2.352h-2.723v2.352H164.4v2.045h1.342v5.114c-.012 1.924 1.298 2.876 3.273 2.793a5.306 5.306 0 001.477-.255l-.428-2.027c-.135.026-.422.09-.678.09-.543 0-.921-.205-.921-.959v-4.756h1.848v-2.045z"/></g><g filter="url(#filter12_d)" fill="#fff"><path d="M133.837 372.492c-.364-3.023-2.653-4.762-5.682-4.762-3.459 0-6.098 2.442-6.098 6.725 0 4.269 2.595 6.724 6.098 6.724 3.355 0 5.382-2.231 5.682-4.641l-2.8-.012c-.262 1.399-1.361 2.205-2.838 2.205-1.988 0-3.336-1.477-3.336-4.276 0-2.723 1.329-4.277 3.355-4.277 1.515 0 2.608.876 2.819 2.314h2.8zM140.127 381.192c2.978 0 4.832-2.039 4.832-5.063 0-3.042-1.854-5.075-4.832-5.075-2.979 0-4.833 2.033-4.833 5.075 0 3.024 1.854 5.063 4.833 5.063zm.013-2.11c-1.375 0-2.078-1.259-2.078-2.972s.703-2.979 2.078-2.979c1.348 0 2.051 1.266 2.051 2.979s-.703 2.972-2.051 2.972zM149.373 375.324c.006-1.266.76-2.007 1.86-2.007 1.093 0 1.751.716 1.745 1.917V381h2.723v-6.251c0-2.289-1.342-3.695-3.388-3.695-1.457 0-2.512.716-2.953 1.86h-.115v-1.732h-2.595V381h2.723v-5.676zM160.49 375.324c.006-1.266.761-2.007 1.86-2.007 1.093 0 1.751.716 1.745 1.917V381h2.723v-6.251c0-2.289-1.342-3.695-3.388-3.695-1.457 0-2.512.716-2.953 1.86h-.115v-1.732h-2.595V381h2.723v-5.676zM173.359 381.192c2.429 0 4.065-1.183 4.449-3.005l-2.519-.166c-.275.748-.978 1.138-1.886 1.138-1.361 0-2.224-.901-2.224-2.365v-.006h6.686v-.748c0-3.337-2.02-4.986-4.615-4.986-2.889 0-4.762 2.052-4.762 5.082 0 3.113 1.847 5.056 4.871 5.056zm-2.18-6.092c.058-1.118.908-2.013 2.116-2.013 1.182 0 2 .843 2.007 2.013h-4.123zM183.998 381.192c2.614 0 4.257-1.534 4.385-3.791h-2.57c-.159 1.049-.85 1.637-1.783 1.637-1.272 0-2.096-1.068-2.096-2.947 0-1.854.83-2.915 2.096-2.915.997 0 1.636.659 1.783 1.636h2.57c-.115-2.269-1.834-3.758-4.398-3.758-2.978 0-4.819 2.065-4.819 5.075 0 2.985 1.809 5.063 4.832 5.063zM195.269 371.182h-1.847v-2.352h-2.724v2.352h-1.342v2.045h1.342v5.114c-.012 1.924 1.298 2.876 3.273 2.793a5.306 5.306 0 001.477-.255l-.428-2.027c-.135.026-.422.09-.678.09-.543 0-.92-.205-.92-.959v-4.756h1.847v-2.045zM208.201 371.674h2.653c-.038-2.333-1.956-3.944-4.871-3.944-2.87 0-4.973 1.585-4.96 3.963-.006 1.931 1.355 3.036 3.567 3.567l1.425.358c1.426.345 2.218.754 2.225 1.636-.007.959-.914 1.611-2.321 1.611-1.438 0-2.473-.665-2.563-1.975h-2.678c.07 2.832 2.096 4.295 5.273 4.295 3.196 0 5.076-1.527 5.082-3.924-.006-2.18-1.649-3.337-3.925-3.848l-1.176-.282c-1.138-.262-2.09-.684-2.071-1.623 0-.844.748-1.464 2.103-1.464 1.323 0 2.135.601 2.237 1.63zM217.2 381.192c2.978 0 4.832-2.039 4.832-5.063 0-3.042-1.854-5.075-4.832-5.075-2.979 0-4.833 2.033-4.833 5.075 0 3.024 1.854 5.063 4.833 5.063zm.012-2.11c-1.374 0-2.077-1.259-2.077-2.972s.703-2.979 2.077-2.979c1.349 0 2.052 1.266 2.052 2.979s-.703 2.972-2.052 2.972zM230.019 376.82c.006 1.316-.895 2.007-1.873 2.007-1.029 0-1.694-.723-1.7-1.88v-5.765h-2.723v6.251c.006 2.295 1.348 3.695 3.323 3.695 1.477 0 2.538-.761 2.979-1.911h.102V381h2.615v-9.818h-2.723v5.638zM234.84 381h2.723v-5.555c0-1.208.882-2.039 2.084-2.039.377 0 .895.064 1.15.147v-2.416a4.117 4.117 0 00-.856-.096c-1.1 0-2.001.639-2.359 1.854h-.102v-1.713h-2.64V381zM246.104 381.192c2.615 0 4.257-1.534 4.385-3.791h-2.57c-.159 1.049-.85 1.637-1.783 1.637-1.272 0-2.096-1.068-2.096-2.947 0-1.854.83-2.915 2.096-2.915.997 0 1.636.659 1.783 1.636h2.57c-.115-2.269-1.834-3.758-4.398-3.758-2.978 0-4.819 2.065-4.819 5.075 0 2.985 1.809 5.063 4.832 5.063zM256.627 381.192c2.429 0 4.065-1.183 4.449-3.005l-2.519-.166c-.274.748-.978 1.138-1.885 1.138-1.362 0-2.225-.901-2.225-2.365v-.006h6.686v-.748c0-3.337-2.02-4.986-4.615-4.986-2.889 0-4.762 2.052-4.762 5.082 0 3.113 1.848 5.056 4.871 5.056zm-2.18-6.092c.058-1.118.908-2.013 2.116-2.013 1.183 0 2.001.843 2.007 2.013h-4.123zM270.999 373.982c-.243-1.809-1.7-2.928-4.18-2.928-2.512 0-4.168 1.163-4.161 3.055-.007 1.471.92 2.423 2.838 2.806l1.7.339c.857.173 1.247.486 1.259.978-.012.582-.645.997-1.598.997-.971 0-1.617-.415-1.783-1.214l-2.678.141c.255 1.879 1.853 3.036 4.455 3.036 2.544 0 4.366-1.298 4.372-3.235-.006-1.419-.933-2.269-2.838-2.659l-1.777-.358c-.914-.198-1.253-.511-1.246-.984-.007-.588.658-.972 1.502-.972.946 0 1.508.518 1.642 1.151l2.493-.153z"/></g><path opacity=".85" fill="url(#paint1_linear)" d="M0 288h398v264H0z"/></g><defs><filter id="filter0_d" x="101" y="389" width="190" height="105" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter1_d" x="101" y="442" width="190" height="105" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter2_d" x="101" y="495" width="190" height="105" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter3_d" x="156.696" y="421.467" width="81.838" height="125.714" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="2"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.27 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter4_d" x="155.037" y="239.776" width="86.925" height="90.056" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter5_d" x="126.312" y="227" width="144.818" height="144.818" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter6_d" x="173.58" y="276.146" width="90.26" height="90.26" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter7_d" x="32" y="75" width="184" height="184" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter8_d" x="176" y="55" width="148" height="148" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter9_dd" x="8" y="143" width="194" height="93" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="3"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0.631373 0 0 0 0 0.937255 0 0 0 1 0"/><feBlend mode="normal" in2="effect1_dropShadow" result="effect2_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect2_dropShadow" result="shape"/></filter><filter id="filter10_dd" x="195" y="77" width="194" height="93" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="10"/><feGaussianBlur stdDeviation="15"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.45 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="3"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0.631373 0 0 0 0 0.937255 0 0 0 1 0"/><feBlend mode="normal" in2="effect1_dropShadow" result="effect2_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect2_dropShadow" result="shape"/></filter><filter id="filter11_d" x="80.675" y="110.161" width="93.817" height="43.031" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="2"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.27 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter12_d" x="118.057" y="364.73" width="157.167" height="21.462" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="1"/><feGaussianBlur stdDeviation="2"/><feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.27 0"/><feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><linearGradient id="paint0_linear" x1="204" y1="28" x2="204" y2="145" gradientUnits="userSpaceOnUse"><stop stop-color="#3E444B" stop-opacity="0"/><stop offset="1" stop-color="#3E444B"/></linearGradient><linearGradient id="paint1_linear" x1="199" y1="288" x2="199" y2="504" gradientUnits="userSpaceOnUse"><stop stop-color="#13171C" stop-opacity="0"/><stop offset="1" stop-color="#13171C"/></linearGradient></defs></svg>
```
This should look the same.
I used SVGOMG with disabled option "Remove unknowns and defaults", to avoid SVG being dimmed.